### PR TITLE
Adds TV

### PR DESCRIPTION
**Pull request by dholloran-ample**
DO NOT MERGE

Code pulled from Contentful this allows us the ability to comment on certain lines

### CODE REVIEW

File: tv2.html [hunk 1]
```diff
@@ -0,0 +1,283 @@
...
+                    <div class="embed-responsive embed-responsive-16by9" id="js-media-video" player-id="player"
+                      video-id="9vmwUS1E-Lc" data-video-player>
+                      <div id="player"></div>
+                    </div>
+                  </div>
+                </div>
+              </a>
+            </div>
+            <h2 class="latest-message-headline font-family-condensed-extra text-uppercase flush-top text-gray-dark">
+              IS GOD GOOD? | THE GOOD GOD JOURNEY WEEK 1
```
Comment (dholloran-ample) [inline tv2.html:24]: I find it is a best practice to make titles title case so you would have "Is God Good? | The Good God Journey Week 1" and then use CSS `text-transform: uppercase;` to force it to uppercase

File: tv2.html [hunk 2]
```diff
@@ -0,0 +1,283 @@
...
+      </div>
+      <div class="col-sm-6 push-bottom">
+        <crds-portrait-card href="https://crossroads.net/media/podcasts/the-aggressive-life-with-brian-tome"
+          theme="default" lead="Listen Now" title="Agressive life podcast"
+          image-src="//crds-media.imgix.net/5APtNFVXDhx8uTbHpUc5sD/f2bb379a5b3ad54681f6a3ec37ff7bc5/bt-agressive-life-with-logos.png">
+        </crds-portrait-card>
+      </div>
+    </div>
+    <div class="col-sm-4 d-flex bt-guide-container"
+      style="background-image: url('//crds-media.imgix.net/TB73SSM1V7gnlY8J9PkEC/75c0303bfede033c1508d4cef22e1a2a/bt-map.png'); background-size: cover;">
```
Comment (dholloran-ample) [inline tv2.html:199]: I would pull this out into a CSS class.

File: tv2.html [hunk 3]
```diff
@@ -0,0 +1,283 @@
...
+      <div class="guide-content">
+        <h2 class="font-family-condensed-extra text-white">YOUR GUIDE TO THE ADVENTUROUS LIFE YOU WERE
```
Comment (dholloran-ample) [inline tv2.html:201]: Use `text-transform: uppercase;` (See above comment)

File: tv2.html [hunk 4]
```diff
@@ -0,0 +1,283 @@
...
+</section>
+
+<div aria-labelledby="Subscribe email form" class="modal fade" id="subscribeModalForm" role="dialog" tabindex="-1">
+  <div class="modal-dialog" role="document">
+    <div class="modal-content">
+      <div class="modal-header hard-bottom"><button aria-label="Close" class="close pull-right" data-dismiss="modal"
+          type="button"><span aria-hidden="true">&times;</span></button>
+      </div>
+      <div class="modal-body text-left hard-bottom">
+        <p class="component-header text-center text-gray-dark push-ends soft-bottom" style="line-height:.9em;">Get
```
Comment (dholloran-ample) [inline tv2.html:226]: Could you pull the `line-height:.9em` into a separate class? Is this really needed?

File: tv2.html [hunk 5]
```diff
@@ -0,0 +1,283 @@
...
+                          <![endif]-->
+          <script charset="utf-8" type="text/javascript" src="//js.hsforms.net/forms/v2.js"></script>
+        </div>
+      </div>
+      <div class="modal-footer">&nbsp;</div>
+    </div>
+  </div>
+</div>
+
+<style>
```
Comment (dholloran-ample) [inline tv2.html:240]: Could this style block be moved to a CSS file? Some declarations seem specific to the page others seem like utility classes. (I tend to not use `<style>` blocks ever unless it is something coming from CMS/JS/Not hardcoded)

File: tv2.html [hunk 6]
```diff
@@ -0,0 +1,283 @@
...
+              <a href="https://www.crossroads.net/media/series/empires">Season 2</a>
+            </h3>
+          </div>
+          <div class="col-sm-4">
+            <div class="embed-container">
+              <a href="https://www.crossroads.net/media/series/real-encounters-with-god-the-jesus-myth">
+                <div class="media-video">
+                  <img class="media-img full-width"
+                    src="https://crds-media.imgix.net/6jBE4tNgbsMjCKBl4UInoU/48b3ad2708be881dd6fc5e797edead52/real-encounters-with-god-jesus-myth.jpg?auto=format,compress&w=750&h=422&fit=crop"
+                    sizes=" {{ site.image_sizes.one_fourth }}" data-optimize-img />
```
Comment (dholloran-ample) [inline tv2.html:107]: Remove the space in `sizes=" {{`

File: tv2.html [hunk 7]
```diff
@@ -0,0 +1,283 @@
...
+            </h3>
+          </div>
+
+          <div class="col-sm-4">
+            <div class="embed-container">
+              <a href="https://www.crossroads.net/media/series/empires">
+                <div class="media-video">
+                  <img class="media-img full-width"
+                    src="https://crds-media.imgix.net/1hqplcLoggKkY6AI0ocue6/d11177354213d5d357c29ecbf8a06826/EMPIRES_Series_slide.jpg?auto=format,compress&w=750&h=422&fit=crop"
+                    sizes=" {{ site.image_sizes.one_fourth }}" data-optimize-img />
```
Comment (dholloran-ample) [inline tv2.html:89]: Remove the space in sizes=" {{

File: tv2.html [hunk 8]
```diff
@@ -0,0 +1,283 @@
...
+            class="text-uppercase font-family-serif font-size-base font-size-small text-gray-light flush-top push-half-sides">
+            real encounters with God
+          </h2>
+          <div class="col-sm-4">
+            <div class="embed-container">
+              <a href="https://www.crossroads.net/media/series/real-encounters-with-god">
+                <div class="media-video">
+                  <img class="media-img full-width"
+                    src="https://crds-media.imgix.net/4290QD6wnvVOL98cPnlmwz/0c6e9d0e07f266a32e0bbcc7690c42e2/Screen_Shot_2020-03-06_at_4.05.46_PM.png?auto=format,compress&w=750&h=422&fit=crop"
+                    sizes=" {{ site.image_sizes.one_fourth }}" data-optimize-img />
```
Comment (dholloran-ample) [inline tv2.html:70]: Remove the space in sizes=" {{

**Comment by silasbaur** [comment]
Hey Jake! Just wanted to give feedback on how it looks on the front-end. Attention to detail is this biggest takeaway from this. If you need help with this lets talk.
1. This does not line up (desktop)
Screen Shot 2020-11-18 at 4 14 55 PM
2. Mobile does not look good here. Looks like you have a lot of unnecessary padding. It looks like you are wrapping everything with a lot of padding classes. 
Screen Shot 2020-11-18 at 4 18 35 PM
Screen Shot 2020-11-18 at 4 24 35 PM

Removing all of the `soft-sides` classes fixes this and it still looks good on desktop
Screen Shot 2020-11-18 at 4 25 17 PM

File: tv2.html [hunk 9]
```diff
@@ -0,0 +1,283 @@
...
+permalink: /tv2
+---
+<section class="bg-tan-light soft-sides soft-top">
+  <div class="soft-sides soft-top">
+    <div class="container soft-top hard-sides">
+      <div class="col-md-8">
+        <div class="latest-message">
+          <div class="latest-message-background soft-half-bottom push-bottom">
+            <div class="embed-container">
+              <a href="#item" data-automation-id="message-video">
```
Comment (dholloran-ample) [inline tv2.html:12]: I changed `href="{{ item | media_url }}"` on this line to `href="#item"` so the build wouldn't fail

**Review by dholloran-ample — COMMENTED** [review]
@jakecatron I left some inline comments let me know if you need clarification on anything.

I did have a general question, what is the reason why this is in Contentful and not in the repo?

(Since I'm the author I can't request changes but I am requesting changes)

File: tv2.html (hunk 3 above)
Comment (silasbaur) [reply tv2.html:201]: Or use the class `text-uppercase`

File: tv2.html (hunk 5 above)
Comment (silasbaur) [reply tv2.html:240]: Agreed. When I am making a page that will end up in the CMS. I always put in the codebase first so if it is absolutely necessary to write more styles for a page then I can put them in the repo. Best thing to do is get as far as you can without writing styles. (we already have so many styles available to us)

File: tv2.html (hunk 1 above)
Comment (jakecatron) [reply tv2.html:24]: @dholloran-ample Yes, good point. I was planning to put this in the repo but wanted to get something over to AG and Sarah for review while I was still working on some stuff with the subscribe modal. So the page in Contentful was really just intended to be something for them to preview quickly while I was finishing up. I hate permanently putting things in Contentful unless I'm explicitly asked to. This is why I was using <style> tags as well just to make it "work" until I was ready to commit everything to the repo. Hope that helps explain some of the mess - sorry you had to weed through that.

**Comment by dholloran-ample** [comment]
Closing in lieu of #1920 being created.

**Pull request closed without merging**